nation_info.rb

```diff
@@ -0,0 +1,37 @@
+class NationInfo
+  def initialize(name,population,gdp,states,army_strength,state_of_country)
+    @name=name
+    @population=population
+    @gdp=gdp
+    @states=states
+    @army_strength=army_strength
+    @state_of_country=state_of_country
+  end
+end
+
+class Questions<NationInfo
+  def loan
```

Comment:
```suggestion
  def loan
      return  'The loan is approved' if @gdp >= 500
      "The loan can't be approved"
    end
```

@Sumit2280 You can use it like this also